File: app/src/main/java/ir/saltech/answersheet/object/data/Device.java
package ir.saltech.answersheet.object.data;

import androidx.annotation.NonNull;

public class Device {
	private int id;
	private String model;
	private String manufacturer;
	private boolean evoked;
	private boolean rejected;
	private boolean accepted;

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	@NonNull
	public String getModel() {
		return model;
	}

	public void setModel(@NonNull String model) {
		this.model = model;
	}

	@NonNull
	public String getManufacturer() {
		return manufacturer;
	}

	public void setManufacturer(@NonNull String manufacturer) {
		this.manufacturer = manufacturer;
	}

	public boolean isEvoked() {
		return evoked;
	}

	public void setEvoked(boolean evoked) {
		this.evoked = evoked;
	}

	public boolean isRejected() {
		return rejected;
	}

	public void setRejected(boolean rejected) {
		this.rejected = rejected;
	}

	public boolean isAccepted() {
		return accepted;
	}

	public void setAccepted(boolean accepted) {
		this.accepted = accepted;
	}

	@NonNull
	@Override
	public String toString() {
		return "Device{" +
				"id=" + id +
				", model='" + model + '\'' +
				", manufacturer='" + manufacturer + '\'' +
				", evoked=" + evoked +
				", rejected=" + rejected +
				", accepted=" + accepted +
				'}';
	}
}

File: app/src/main/java/ir/saltech/answersheet/object/data/Device.kt
package ir.saltech.answersheet.`object`.data

class Device {
    var id: Int = 0
    private var model: String? = null
    private var manufacturer: String? = null
    var isEvoked: Boolean = false
    var isRejected: Boolean = false
    var isAccepted: Boolean = false

    fun getModel(): String {
        return model!!
    }

    fun setModel(model: String) {
        this.model = model
    }

    fun getManufacturer(): String {
        return manufacturer!!
    }

    fun setManufacturer(manufacturer: String) {
        this.manufacturer = manufacturer
    }

    override fun toString(): String {
        return "Device{" +
                "id=" + id +
                ", model='" + model + '\'' +
                ", manufacturer='" + manufacturer + '\'' +
                ", evoked=" + isEvoked +
                ", rejected=" + isRejected +
                ", accepted=" + isAccepted +
                '}'
    }
}
